controllers/install/lookout_controller.go

```diff
@@ -49,6 +49,7 @@ type LookoutReconciler struct {
 
 //+kubebuilder:rbac:groups=install.armadaproject.io,resources=lookouts,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=install.armadaproject.io,resources=lookouts/status,verbs=get;update;patch
+//+kubebuilder:rbac:groups="batch",resources=jobs,verbs=get;list;watch;create;delete;deletecollection;patch;update
```

Comment:
What caused this change?

Reply:
Now that we are starting jobs by default, the rbac was complaining that it didn't have permission to actually create the batch jobs.  So I added it here so now we are able to actually create those jobs.  

You can find this by looking in the logs of an operator that is deployed to Kind.